Sources/PrimerSDK/Classes/Error Handler/PrimerError.swift

```diff
@@ -485,6 +488,13 @@ enum PrimerError: PrimerErrorProtocol {
                                      bundle: Bundle.primerResources,
                                      value: "Amount cannot be null for order items with isPending == false",
                                      comment: "Amount cannot be null for order items with isPending == false - Primer error message")
+            
+        case .intentNotSupported(let intent, let paymentMethodType):
+            return NSLocalizedString("primer-error-message-intent-not-supported",
+                                     tableName: nil,
+                                     bundle: Bundle.primerResources,
+                                     value: "Intent is not supported",
+                                     comment: "Intent is not supported - Primer error message") + "\(paymentMethodType.rawValue).\(intent.rawValue)"
```

Comment:
I know it's an edge case but do you think it makes sense to call it just `Intent`, I believe it's better to call `PrimerSessionIntent not supported`?

Reply:
Done